src/Microsoft.ML.Core/Data/ColumnTypeExtensions.cs

```diff
@@ -17,5 +17,10 @@ internal static class ColumnTypeExtensions
         public static bool IsStandardScalar(this ColumnType columnType) =>
             (columnType is NumberType) || (columnType is TextType) || (columnType is BoolType) ||
             (columnType is TimeSpanType) || (columnType is DateTimeType) || (columnType is DateTimeOffsetType);
+
+        /// <summary>
+        /// Zero return means either it's not a key type or the cardinality is unknown.
+        /// </summary>
+        public static int KeyCount(this ColumnType columnType) => (columnType as KeyType)?.Count ?? 0;
```

Comment:
Thoughts on whether this should be `GetKeyCount()` instead? It's an internal API, so we can rename it at any time...

Reply:
Since you solicited an opinion a verbified name would probably be best. Even for internal APIs, it might be best to set good practices, since I notice bad practices have a way of spreading. 😄 But given that your PR is ready *right now* to check in, I'd be inclined to not bother...

Reply:
I'll fix it - it is a simple "Rename" in VS, so very little work.